WooCommerce/Classes/ViewRelated/Dashboard/Settings/About/WooAboutScreenConfiguration.swift

```diff
@@ -0,0 +1,226 @@
+        static let website          = URL(string: "https://woocommerce.app/")!
+        static let blog             = URL(string: "https://woocommerce.com/blog/")!
+        static let automatticFamily = URL(string: "https://automattic.com")!
+        static let workWithUs       = URL(string: "https://automattic.com/work-with-us")!
+    }
+
+    enum Localization {
+        enum Titles {
+            static let rateUs           = NSLocalizedString("Rate us", comment: "Title of button that allows the user to rate the app in the App Store")
+            static let share             = NSLocalizedString("Share with Friends", comment: "Title for button allowing users to share "
+                                                             + "information about the app with friends, such as via Messages")
+            static let instagram        = NSLocalizedString("Instagram", comment: "Title of a button linking to the app's Instagram profile")
+            static let twitter          = NSLocalizedString("Twitter", comment: "Title of a button linking to the app's Twitter profile")
+            static let website          = NSLocalizedString("Website", comment: "Title of a button linking to the app's website")
+            static let blog             = NSLocalizedString("Blog", comment: "Title of a button linking to the app's blog")
+            static let legalAndMore     = NSLocalizedString("Legal and more", comment: "Title of a button linking to a list of legal documents "
+                                                            + "like privacy policy,terms of service, etc")
+            static let automatticFamily = NSLocalizedString("Automattic family", comment: "Title of a button linking to the Automattic website")
+            static let workWithUs       = NSLocalizedString("Work with us", comment: "Title of a button linking to the Automattic Work With Us web page")
+        }
+
+        enum Subtitles {
+            static let website          = "woocommerce.app"
+            static let blog             = "woocommerce.com/blog"
+            static let twitter          = "@woocommerce"
+            static let workWithUs       = NSLocalizedString("Join from anywhere",
+                                                            comment: "Subtitle for button displaying the Automattic Work With Us web page, "
+                                                            + "indicating that Automattic employees can work from anywhere in the world")
+        }
+
+        static let shareSheetMessage = NSLocalizedString("Hey! Here is a link to download the WooCommerce app. "
+                                                         + "I'm really enjoying it and thought you might too.",
+                                                         comment: "Text used when sharing a link to the app with a friend.")
+    }
+}
+
+// MARK: - Legal and More submenu
+
+class WooLegalAndMoreSubmenuConfiguration: AboutScreenConfiguration {
```

Comment:
nit: could this be `final`?
```suggestion
final class WooLegalAndMoreSubmenuConfiguration: AboutScreenConfiguration {
```